src/Components/RegisterForm.js

```diff
@@ -44,25 +86,31 @@ class RegisterForm extends Component {
         <header className="user__header">
             <h1 className="user__title">Sign-up</h1>
         </header>
-        <form className="form" onSubmit = {this.createUser}>
+        <form className="form" onSubmit={this.createUser}>
             <div className="form__group">
-                <input type="text"  placeholder="Username" className="form__input" onChange = {(event) => this.handleInputChange(event, 'userName')}/>
+                <input type="text"  placeholder="Username" className="form__input" onChange={(event) => this.handleInputChange(event, 'userName')}/>
             </div>
 
             <div className="form__group">
-                <input type="email"  placeholder="Email" className="form__input" onChange = {(event) => this.handleInputChange(event, 'email')}/>
+                <input type="email"  placeholder="Email" className="form__input" onChange={(event) => this.handleInputChange(event, 'email')}/>
             </div>
 
             <div className="form__group">
-                <input type="password"  name="password" placeholder="Password" className="form__input" onChange = {(event) => this.handleInputChange(event, 'password')}/>
+                <input type="password"  name="password" placeholder="Password" className="form__input" onChange={(event) => this.handleInputChange(event, 'password')}/>
             </div>
 
             <div className="form__group">
-                <input type="password"  name="cPassword" placeholder="Confirm password" className="form__input" onChange = {(event) => this.handleInputChange(event, 'cPassword')}/>
+                <input type="password"  name="cPassword" placeholder="Confirm password" className="form__input" onChange={(event) => this.handleInputChange(event, 'cPassword')}/>
             </div>
             <button className="btn"  type="submit">Register</button>
             {this.props.registerSuccess && <Redirect to='/Login'/>}
         </form>
+        <div>
+          <p>{this.state.userName.error}</p>
+          <p>{this.state.email.error}</p>
+          <p>{this.state.password.error}</p>
+          <p>{this.state.cPassword.error}</p>
```

Comment:
provide client validation messages under each input and server validation messages such as "User already exist" under the whole form